### PR TITLE
Fixes #16792: When upgrading to 6.0.3 the packaging should disable all installed plugins

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/16792

### CODE REVIEW

File: rudder-webapp/SOURCES/rudder-upgrade [hunk 1]
```diff
@@ -532,7 +532,10 @@ EOF
 # Check and upgrade plugins
 ################################################################################
 upgrade_plugins() {
-  rudder package check-compatibility
+  # For 6.0.3 only we disable all installed plugins at install time since they
+  # are not compatible
+  #rudder package check-compatibility
```
Comment (ncharles) [inline rudder-webapp/SOURCES/rudder-upgrade]: you should add a message there, for the user to know

**Comment by Fdall** [comment]
PR updated with a new commit

File: rudder-webapp/SOURCES/rudder-webapp-postinst [hunk 2]
```diff
@@ -166,8 +166,20 @@ echo "Done"
...
 if [ -f /tmp/rudder-plugins-upgrade ]
 then
-  rudder package plugin restore-status < /tmp/rudder-plugins-upgrade || true
-  rudder package check-compatibility || true
+  # For 6.0.3 only we disable all installed plugins at install time since they
+  # are not compatible
+  #rudder package plugin restore-status < /tmp/rudder-plugins-upgrade || true
+  #rudder package check-compatibility || true
+  echo "********************************************************************************"
+  echo "Due to incompatibility reasons for this specific version all plugins have been
```
Comment (Fdall) [inline rudder-webapp/SOURCES/rudder-webapp-postinst]: typo

Comment (VinceMacBuche) [reply rudder-webapp/SOURCES/rudder-webapp-postinst]: fix it instead of commenting it

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
OK, squash merging this PR

**Pull request merged**